niworkflows/anat/ants.py

```diff
@@ -310,7 +311,7 @@ def init_brain_extraction_wf(name='brain_extraction_wf',
             ('reverse_transforms', 'transforms'),
             ('reverse_invert_flags', 'invert_transform_flags')]),
         (map_brainmask, thr_brainmask, [('output_image', 'input_image')]),
-        (thr_brainmask, dil_brainmask, [('output_image', 'op1')]),
+        (thr_brainmask, dil_brainmask, [('out_mask', 'op1')]),
```

Comment:
```suggestion
        (thr_brainmask, dil_brainmask, [('output_image', 'op1')]),
```

Reply:
Are you sure? That connection wasn't working, which is why I changed it.

Reply:
Ah, I see the confusion. We are using `Binarize`, which has an output `out_mask` not `output_image` (that's if we were using the ants function, but we're not).